test/parallel/test-module-create-require.js

```diff
@@ -5,8 +5,17 @@ const assert = require('assert');
 const path = require('path');
 
 const { createRequireFromPath } = require('module');
+const { createRequire } = require('module');
```

Comment:
Maybe add a test for what happens with invalid input? (like a 'junk' string as URL)

Reply:
@benjamingr a garbage string is not "invalid input" afaict based on the implementation of `createRequireFromPath`. Things would fail later when using the generated `require`, but this behavior is specific to Module and not createRequire. I personally don't think we need extra tests TBQH

Reply:
more tests added